### PR TITLE
[SEM - 008] : SecondaryButton

### PR DESCRIPTION
**Pull request by armandogear5**
https://ditmar.github.io/backoffice-openscience/branch-SEM-008/?path=/story/ui-components-atoms-secondarybutton

### CODE REVIEW

File: src/ui/components/utils/svg-icons/css/grid-icons.module.scss [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  
-  .item {
-    box-sizing: border-box; // Incluye el padding dentro del ancho
-    flex: 1 1 calc(10% - 10px); // Ocupa un tercio del contenedor menos el espacio del gap
-    min-width: 100px; // Asegura que los elementos no se hagan más pequeños de lo debido
-    padding: 10px;
-  }
\ No newline at end of file
+  display: flex;
+  flex-wrap: wrap; // Permite que los elementos se envuelvan a la siguiente línea
```
Comment (Ditmar) [inline src/ui/components/utils/svg-icons/css/grid-icons.module.scss:3]: remove the GPT comments

File: src/ui/components/utils/svg-icons/css/grid-icons.module.scss [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  gap: 20px; // Espacio entre los elementos, similar a `grid-gap` en CSS Grid
```
Comment (Ditmar) [inline src/ui/components/utils/svg-icons/css/grid-icons.module.scss:4]: remove the GPT comments

File: src/ui/components/utils/svg-icons/css/grid-icons.module.scss [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+  justify-content: space-between; // Distribuye los elementos con espacio entre ellos
```
Comment (Ditmar) [inline src/ui/components/utils/svg-icons/css/grid-icons.module.scss:5]: remove the GPT comments

File: src/ui/components/atoms/SecondaryButton/SecondaryButton.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+function SecondaryButton({
+  size = 'medium',
+  disabled = false,
+  onClick,
+  active = false,
+  focused = false,
+  hovered = false,
+}: SecondaryButtonProps) {
+  const [isHovered, setIsHovered] = useState(hovered);
```
Comment (Ditmar) [inline src/ui/components/atoms/SecondaryButton/SecondaryButton.tsx:14]: Why you need this states ?

**Review by Ditmar — CHANGES_REQUESTED** [review]
Read the comments

**Comment by Ditmar** [comment]
incorrect font, please check the figma 
image

File: src/ui/components/atoms/SecondaryButton/SecondaryButton.tsx (hunk 4 above)
Comment (armandogear5) [reply src/ui/components/atoms/SecondaryButton/SecondaryButton.tsx:14]: Using these states allows you to create a more interactive and accessible button component that responds to different user interactions from both mouse and keyboard events. 
These states are useful for applying different CSS styles using the generated class (classNames).

**Comment by armandogear5** [comment]
https://github.com/Ditmar/backoffice-openscience/tree/feature/SEM-008/src/ui/components/atoms/SecondaryButton

**Comment by armandogear5** [comment]
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-008/

**Comment by armandogear5** [comment]
Please I want a review of my pr
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-008/?path=/story/ui-components-atoms-secondarybutton

**Comment by DanX3N0** [comment]
Check the demo structure and apply it to your ticket:
{609E32AF-56BD-4B2F-A8D7-F8856853A3CC}
{4125604F-29EC-40BD-AFF5-F5CD3FE19FB6}

**Comment by armandogear5** [comment]
Check the demo structure and apply it to your ticket:
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-008/?path=/story/ui-components-atoms-secondarybutton--enabled

**Comment by armandogear5** [comment]
Check the demo structure and apply it to your ticket:
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-008/?path=/story/ui-components-atoms-secondarybutton--enabled

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Comment by armandogear5** [comment]
Please can you check my ticket?
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-008/?path=/story/ui-components-atoms-secondarybutton--small

**Review by Ditmar — APPROVED** [review]

**Pull request merged**